jujugui/static/gui/src/app/components/profile/profile.js

```diff
@@ -195,20 +255,26 @@ class Profile extends React.Component {
       <Panel
         instanceName="profile"
         visible={true}>
-        <ProfileHeader
-          changeState={props.changeState}
-          controllerIP={props.controllerIP}
-          getUser={props.getUser}
-          gisf={props.gisf}
-          userInfo={shapeup.fromShape(props.userInfo, ProfileHeader.propTypes.userInfo)} />
-        <div className="twelve-col">
-          <div className="profile__content inner-wrapper">
-            <ProfileNavigation
-              // Use supplied activeSection or the key from the first map entry.
-              activeSection={this._getSectionInfo().active || mapEntry[0]}
-              changeState={props.changeState}
-              sectionsMap={sectionsMap} />
-            {section.getComponent()}
+        <div className="v1">
```

Comment:
we can pass extra classes to the Panel, can you do that instead of adding a wrapper div?